timer/README.md

```diff
@@ -0,0 +1,15 @@
+Generic timer multiplexer
```

Comment:
Is the timer code meant to be in this PR?

Reply:
I think it's just my fork is behind and therefore pulled in timer.
I'm assuming at merge time it'll see it's already there and get ignored...